Makefile

```diff
@@ -40,7 +40,7 @@ generate-client:
 	$(call run_generator,-DsupportingFiles=README.md)
 
 .PHONY: validate
-validate: test
+validate: generate
```

Comment:
The point of validate running test too was to have a single target to run in CI, without needing to generate stuff twice (both validate and test require generate).